### PR TITLE
HDDS-1231. Add ChillMode metrics.

### PR DESCRIPTION
**Pull request by bharatviswa504**

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 36 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 1 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 537 | trunk passed |
| +1 | compile | 267 | trunk passed |
| +1 | checkstyle | 78 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 931 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 148 | trunk passed |
| 0 | spotbugs | 292 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 485 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 473 | the patch passed |
| +1 | compile | 270 | the patch passed |
| +1 | javac | 270 | the patch passed |
| -0 | checkstyle | 40 | hadoop-hdds: The patch generated 2 new + 0 unchanged - 0 fixed = 2 total (was 0) |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 750 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 147 | the patch passed |
| +1 | findbugs | 499 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 165 | hadoop-hdds in the patch failed. |
| -1 | unit | 1140 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 49 | The patch does not generate ASF License warnings. |
| | | 11627 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.web.client.TestKeys |
|   | hadoop.ozone.client.rpc.TestBCSID |
|   | hadoop.ozone.container.common.impl.TestContainerPersistence |
|   | hadoop.hdds.scm.pipeline.TestSCMPipelineManager |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-851/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/851 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux aac9b558311c 4.4.0-143-generic #169~14.04.2-Ubuntu SMP Wed Feb 13 15:00:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 55e0c13 |
| Default Java | 1.8.0_212 |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/1/artifact/out/diff-checkstyle-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/1/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/1/testReport/ |
| Max. process+thread count | 4422 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/server-scm U: hadoop-hdds/server-scm |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/1/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/SafeModeMetrics.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package org.apache.hadoop.hdds.scm.safemode;
```
Comment (jiwq) [inline hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/SafeModeMetrics.java]: License missing

Comment (bharatviswa504) [reply hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/SafeModeMetrics.java]: Fixed.

**Comment by bharatviswa504** [comment]
Thank You for the review @jiwq.
Fixed review comments and also fixed other checkstyle issues.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 473 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 537 | trunk passed |
| +1 | compile | 264 | trunk passed |
| +1 | checkstyle | 76 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 937 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 150 | trunk passed |
| 0 | spotbugs | 296 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 486 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 478 | the patch passed |
| +1 | compile | 267 | the patch passed |
| +1 | javac | 267 | the patch passed |
| +1 | checkstyle | 80 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 735 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 147 | the patch passed |
| +1 | findbugs | 512 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 168 | hadoop-hdds in the patch failed. |
| -1 | unit | 1232 | hadoop-ozone in the patch failed. |
| -1 | asflicense | 45 | The patch generated 17 ASF License warnings. |
| | | 6706 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestWatchForCommit |
|   | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.ozone.client.rpc.TestBCSID |
|   | hadoop.ozone.client.rpc.TestFailureHandlingByClient |
|   | hadoop.ozone.container.common.impl.TestContainerPersistence |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-851/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/851 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 4269c84dcf75 4.4.0-139-generic #165~14.04.1-Ubuntu SMP Wed Oct 31 10:55:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 83549db |
| Default Java | 1.8.0_212 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/3/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/3/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/3/testReport/ |
| asflicense | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/3/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 4630 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/server-scm U: hadoop-hdds/server-scm |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/3/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by anuengineer** [comment]
+1, Thanks for the contribution.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 0 | Docker mode activated. |
| -1 | patch | 14 | https://github.com/apache/hadoop/pull/851 does not apply to trunk. Rebase required? Wrong Branch? See https://wiki.apache.org/hadoop/HowToContribute for help. |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/851 |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-851/4/console |
| versions | git=1.9.1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**